Dockerfile

```diff
@@ -12,11 +12,15 @@ RUN cd frontend && npm run bundledjango
 
 
 # Step 2 - Build Django Application
-FROM python:3.9-slim as application
+FROM python:3.10-slim as application
 
 WORKDIR /app
 COPY api /app/
 
+# arm architecture platform builds need postgres drivers installing via apt #todo: can be removed possibly at some point
```

Comment:
This todo is not really helpful, it just adds unnecessary noise and confusion. Can you remove but leave the preceding comment (which _is_ helpful).